package.json

```diff
@@ -14,7 +14,7 @@
     "rebuild": "npm run clean && npm clean-install && npm run build",
     "test": "nx run-many --target=test --exclude=integration-tests,@taquito/website,taquito-test-dapp-vite --collectCoverage",
     "lint": "lerna run lint",
-    "clean": "rm -rf ./node_modules/* && lerna clean --yes",
```

Comment:
`rm -rf ./node_modules/*` is required since `lerna clean` only deletes `node_modules` from sub-packages but not from the root directory.

Reply:
That's true but lerna depends on node_modules. Here's what I'm seeing from a clean clone of master and explicitly running the global lerna by specifying the full path. Can you reproduce? 

```
❯ /home/rich/.config/nvm/versions/node/v16.19.1/bin/lerna clean --yes
node:internal/modules/cjs/loader:1024
  throw err;
  ^

Error: Cannot find module 'nx/presets/npm.json'
Require stack:
- /home/rich/.config/nvm/versions/node/v16.19.1/lib/node_modules/lerna/node_modules/nx/src/config/workspaces.js
- /home/rich/.config/nvm/versions/node/v16.19.1/lib/node_modules/lerna/node_modules/nx/src/devkit-exports.js
- /home/rich/.config/nvm/versions/node/v16.19.1/lib/node_modules/lerna/node_modules/@nrwl/devkit/index.js
- /home/rich/.config/nvm/versions/node/v16.19.1/lib/node_modules/lerna/dist/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1021:15)
    at Function.resolve (node:internal/modules/cjs/helpers:114:19)
    at Workspaces.readNxJson (/home/rich/.config/nvm/versions/node/v16.19.1/lib/node_modules/lerna/node_modules/nx/src/config/workspaces.js:144:52)
    at readNxJson (/home/rich/.config/nvm/versions/node/v16.19.1/lib/node_modules/lerna/node_modules/nx/src/config/configuration.js:7:72)
    at Object.<anonymous> (/home/rich/.config/nvm/versions/node/v16.19.1/lib/node_modules/lerna/node_modules/nx/src/daemon/client/client.js:288:72)
    at Module._compile (node:internal/modules/cjs/loader:1191:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1245:10)
    at Module.load (node:internal/modules/cjs/loader:1069:32)
    at Function.Module._load (node:internal/modules/cjs/loader:904:12)
    at Module.require (node:internal/modules/cjs/loader:1093:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/rich/.config/nvm/versions/node/v16.19.1/lib/node_modules/lerna/node_modules/nx/src/config/workspaces.js',
    '/home/rich/.config/nvm/versions/node/v16.19.1/lib/node_modules/lerna/node_modules/nx/src/devkit-exports.js',
    '/home/rich/.config/nvm/versions/node/v16.19.1/lib/node_modules/lerna/node_modules/@nrwl/devkit/index.js',
    '/home/rich/.config/nvm/versions/node/v16.19.1/lib/node_modules/lerna/dist/cli.js'
  ]
}
```

Reply:
In nx workspaces.js, the following throws
```js
const extendedNxJsonPath = require.resolve(nxJsonConfiguration.extends, {
    paths: [(0, path_1.dirname)(nxJson)],
});
```

The request and options in my case are:
```js
{
  request: 'nx/presets/npm.json',
  options: { paths: [ '/home/rich/tmp/taquito' ] }
}
```
So it's trying to resolve `/home/rich/tmp/taquito/node_modules/nx/presets/npm.json` but it doesn't exist because it was just removed so it throws.

I'm not familiar with `nx` but looking at the configuration, we could probably change the `extends` from `nx/presets/npm.json` to something local install of the packaged version.
```json
{
  "extends": "nx/presets/npm.json",
  "tasksRunnerOptions": {
    "default": {
      "runner": "nx/tasks-runners/default",
      "options": {
        "cacheableOperations": ["build", "example", "test", "lint", "docusaurus"],
        "accessToken": "placeholder"
      }
    }
  },
  "targetDefaults": {
    "build": {
      "dependsOn": ["^build"]
    }
  },
  "affected": {
    "defaultBase": "master"
  }
}
```

Reply:
This is an unrelated bugfix so I can revert it and open a new PR. Let me know what you prefer.

Reply:
>That's true but lerna depends on node_modules. Here's what I'm seeing from a clean clone of master and explicitly running the global lerna by specifying the full path. Can you reproduce?

ah! I have `lerna` installed globally which is why I never faced that error. Thanks for pointing that out. We could probably stop using `lerna clean` altogether and instead use the command [`git clean -fdx`](https://git-scm.com/docs/git-clean/en) to get a clean working directory.